### PR TITLE
Add kad merge, resolve_aliad, sha256_string

### PR DESCRIPTION
**Pull request by folex**

### CODE REVIEW

File: builtin.aqua [hunk 1]
```diff
@@ -97,6 +101,14 @@ service Kademlia("kad"):
...
     -- in the Kademlia neighborhood for a given key
     neighborhood(key: PeerId, already_hashed: bool) -> []PeerId
+    -- Merges given lists and sorts them by distance to target
+    -- Arguments:
+    -- target – base58 string; result is sorted by XOR distance to target
+    -- left – list of base58 strings
+    -- right – list of base58 strings
+    -- count – how many items to return
+    -- Returns: list of base58 strings sorted by distance to target; list will contain at most count elements
+    merge(target: string, left: []string, right: []string, count: u32) -> []string
```
Comment (DieMyst) [inline builtin.aqua:111]: can I use `0` count or something like this for `all`?

File: builtin.aqua [hunk 2]
```diff
@@ -66,6 +66,10 @@ service Op("op"):
     string_from_b58(b: string) -> string
     bytes_to_b58(bs: []u8) -> string
     bytes_from_b58(b: string) -> []u8
+    -- Applies SHA256 to the given string
+    -- Argument: s - string to apply sha256 to
+    -- Returns: returns sha256 multihash encoded as base58
+    sha256_string(s: stribg) -> string
```
Comment (DieMyst) [inline builtin.aqua:72]: typo stribg -> string

File: builtin.aqua (hunk 1 above)
Comment (folex) [reply builtin.aqua:111]: no 🤔

Comment (folex) [reply builtin.aqua:111]: If you omit this argument (in AIR), then the default would be 20. Maybe add this to doc?

Comment (DieMyst) [reply builtin.aqua:111]: for sure

**Review by DieMyst — APPROVED** [review]

**Pull request merged**